waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -457,9 +457,42 @@ proc query*(w: WakuStore, query: HistoryQuery, handler: QueryHandlerFunc) {.asyn
     return
 
   waku_store_messages.set(response.value.response.messages.len.int64, labelValues = ["retrieved"])
-
   handler(response.value.response)
 
+  
+proc findLastSeen*(list: seq[IndexedWakuMessage]): float = 
+  var lastSeenTime = float64(0)
+  for iwmsg in list.items : 
+    lastSeenTime = if iwmsg.msg.timestamp>lastSeenTime: iwmsg.msg.timestamp else: lastSeenTime 
+  return lastSeenTime
+
+proc resume*(ws: WakuStore){.async, gcsafe.} =
```

Comment:
```suggestion
proc resume*(ws: WakuStore) {.async, gcsafe.} =
```

Reply:
Can we add a more detailed docstring to this? Right now it isn't obvious what one would expect this call to do, or what has to be true for it to work properly, etc.

Reply:
sure, done!